webapp/views.py

```diff
@@ -113,7 +113,28 @@ def _get_resources(self):
         content = self.request.GET.get('content')
         page = int(self.request.GET.get('page', 1))
         offset = (page - 1) * self.PER_PAGE
+        search = self.request.GET.get('q')
         feed_items = {}
+        if search:
+            api_url = (
+                    '{api_url}/posts?_embed'
+                    '&search={search}&{resource_filter}'
+                )
+            api_url = api_url.format(
+                    api_url=self.API_URL,
+                    search=search,
+                    resource_filter=self.RESOURCE_FILTER,
+                )
```

Comment:
all this block has an indentation problem

Reply:
```python
api_url = (
                    '{api_url}/posts?_embed'
                    '&search={search}&{resource_filter}'
                ).format(
                    api_url=self.API_URL,
                    search=search,
                    resource_filter=self.RESOURCE_FILTER,
                )
```
instead of instanciating 2 times the variable

Reply:
Done.